pkg/cmd/pulumi/package_gen_sdk.go

```diff
@@ -47,6 +50,31 @@ func newGenSdkCommand() *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			if language == "java" || language == "all" {
+				javaPackageInfo, ok := pkg.Language["java"].(javagen.PackageInfo)
+				if !ok {
+					javaPackageInfo = javagen.PackageInfo{}
+				}
+				javaPackageInfo.BuildFiles = javaBuildTool
```

Comment:
Why can't the SDKs themselves set this? Why do we even need a flag to change build tool, and if we do have a flag why is it set in the schema as well? 
I might just be showing my Java naivete here but it seems odd.

Reply:
This actually solves my issue.